app/controllers/idv/welcome_controller.rb

```diff
@@ -31,9 +30,9 @@ def update
     def self.step_info
       Idv::StepInfo.new(
         key: :welcome,
-        controller: controller_name,
+        controller: self,
         next_steps: [:agreement],
-        preconditions: ->(idv_session:, user:) { true },
+        preconditions: ->(idv_session:, user:) { !user.gpo_verification_pending_profile? },
```

Comment:
This keeps us from falling into the welcome step when the user wants to resend a letter.